### DISCUSSION TITLE
Release Camera?

### DISCUSSION THREAD

**Discussion by chazhenry** [airsdk/Adobe-Runtime-Support#1073] [General]
Successfully using the webcam on BigSur 11.1 and Air 33.1.1.554 under Windows. Question is how to release it when done?

I removeChild from main window to stop showing. I video.clear(). I set the camera object to null, but don't see any methods to release it. My app is often run under Zoom and I need to share the camera. Currently, once I grab it, the light never goes off. I never let it go. Thanks for any help.

**Comment by itlancer** [comment]
Have you done
`video.attachCamera(null);`
?

**Reply by chazhenry** [reply]
I have now. Bingo. Many thanks!
